README.md

```diff
@@ -21,13 +21,12 @@
 
 -->
 
-> # Template
+> # Math_Magicians
 
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, to ensure your documentation is professional, please always double-check that all links are working. Currently, the link to your issues page in the contribution section is broken. Please consider fixing this! 🤓 🔧

Reply:
Hi @anagudelogu 👋,
Thank you so much 🙏  for reviewing my project and giving constructive feedback.
I have ✔️ and make it fix.